ui_testing/testcases/basic_tests/test06_orders.py

```diff
@@ -2238,3 +2238,29 @@ def test036_Duplicate_sub_order_with_multiple_testplans_and_testunits_delet_appr
         self.assertIn(duplicated_suborder_data['Test Units'], test_units)
         self.assertIn(duplicated_suborder_data['Test Plans'], test_plans)
 
+    def test036_duplicate_main_order_with_testPlan_and_testUnit_change_both(self):
+        """
+        Duplicate from the main order Approach: Duplicate then change the test units & test plans
+
+        LIMS-6221
+        """
+        self.info('create order with test plan and test unit')
+        response, payload = self.orders_api.create_new_order()
+        self.orders_page.filter_by_order_no(payload[0]['orderNo'])
+        self.orders_page.duplicate_main_order_from_order_option()
+        self.info("get new test plan and test unit")
+        new_test_plan = random.choice(
+            TestPlanAPI().get_completed_testplans_with_material_and_same_article(
+                material_type=payload[0]['materialType']['text'], article=payload[0]['article']['text']))
+        new_test_unit = TestPlanAPI().get_testplan_form_data(id=new_test_plan['id'])['specifications'][0]['name']
+        new_suborder = self.order_page.update_suborder(
+            test_plans=[new_test_plan['testPlanName']], test_units=[new_test_unit], remove_old=True)
+        self.order_page.save(save_btn='order:save')
+        self.order_page.wait_until_page_is_loaded()
+        self.order_page.get_orders_page()
+        self.assertTrue(self.orders_page.is_order_in_table(new_suborder['orderNo']))
+        self.analyses_page.search(new_suborder['orderNo'])
+        self.order_page.wait_until_page_is_loaded()
```

Comment:
There is no `please wait` action after press on save, so here use `sleep` methods till I found a nice solution
![image](https://user-images.githubusercontent.com/17981089/80650176-c76e7b00-8a73-11ea-844b-9a643b5a6b18.png)

Reply:
ok, thank you